Installation/installation.json

```diff
@@ -1,162 +1,174 @@
 {
-    "installationService": "CommonGateway\\XxllncZGWBundle\\Service\\InstallationService",
-    "applications": [
-      {
-        "title": "Xxllnc Application",
-        "$id": "https://docs.commongateway.nl/application/default.application.json",
-        "$schema": "https://docs.commongateway.nl/schemas/Application.schema.json",
-        "version": "0.0.2",
-        "description": "The xxllnc application for extra configuration",
-        "domains": [
-          "localhost",
-          "api.xxllnc.commonground.nu"
-        ],
-        "configuration": [
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/ZaakType.endpoint.json": {
-              "in": {
-                "query": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncZaaktypeQueryInMapping.mapping.json"
-                }
-              }
-            }
-          },
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/Eigenschap.endpoint.json": {
-              "in": {
-                "query": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncEigenschapQueryInMapping.mapping.json"
-                }
-              }
-            }
-          },
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/InformatieObjectType.endpoint.json": {
-              "in": {
-                "query": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncInformatieObjecttypeQueryInMapping.mapping.json"
-                }
-              }
-            }
-          },
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/ResultaatType.endpoint.json": {
-              "in": {
-                "query": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncResultaattypeQueryInMapping.mapping.json"
-                }
-              }
-            }
-          },
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/BesluitType.endpoint.json": {
-              "in": {
-                "query": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncBesluittypeQueryInMapping.mapping.json"
-                }
+  "installationService": "CommonGateway\\XxllncZGWBundle\\Service\\InstallationService",
+  "applications": [
+    {
+      "title": "Xxllnc Application",
+      "$id": "https://docs.commongateway.nl/application/default.application.json",
```

Comment:
This will work. 
But keep in mind that if multiple bundles do this for the same default reference (and with different versions) they will collide.
Might be a better idea to create a new Application with a unique reference instead 🤔

Reply:
This part of the file hasn't really changed except for some whitespacing